cmd/namespace/use.go

```diff
@@ -0,0 +1,144 @@
+// Copyright 2021 The Okteto Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package namespace
+
+import (
+	"context"
+	"fmt"
+
+	contextCMD "github.com/okteto/okteto/cmd/context"
+	"github.com/okteto/okteto/cmd/utils"
+	"github.com/okteto/okteto/pkg/analytics"
+	"github.com/okteto/okteto/pkg/errors"
+	"github.com/okteto/okteto/pkg/log"
+	"github.com/okteto/okteto/pkg/okteto"
+	"github.com/spf13/cobra"
+)
+
+const (
+	newNamespaceOption = "Create new namespace"
+)
+
+// Namespace fetch credentials for a cluster namespace
+func Use(ctx context.Context) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "use [namespace]",
```

Comment:
Can we have a flag or an envar to default to the personal namespace? I think we will need it to adapt the github namespace action

Reply:
True, I will create an issue on Namespace so we don't break it after merging this PR